build/Dockerfile.nginx

```diff
@@ -7,8 +7,8 @@ ARG BUILD_AGENT
 
 RUN apk add --no-cache libcap \
     && mkdir -p /var/lib/nginx /usr/lib/nginx/modules \
-    && setcap 'cap_net_bind_service=+ep' /usr/sbin/nginx \
-    && setcap -v 'cap_net_bind_service=+ep' /usr/sbin/nginx \
+    && setcap 'cap_net_bind_service=+ep' /usr/sbin/nginx-debug \
```

Comment:
we need to run the command for both `nginx` and `nginx-debug`

Reply:
oh by your comment of 

>can we run them for the debug binary too by default?

I interpreted that by the default would be `nginx-debug`, instead of having both setcap commands, I'll make that change

Reply:
oh yeah I meant to run the `setcap` command for both binaries, sorry for the confusion!